src/watchdog.rs

```diff
@@ -125,21 +125,21 @@ impl IndependentWatchDog {
     #[must_use]
     pub fn interval(&self) -> Milliseconds {
         // If the prescaler was changed wait until the change procedure is finished.
-        while self.iwdg.sr.read().pvu().bit() {}
+        while self.iwdg.sr().read().pvu().bit() {}
 
-        let psc = self.iwdg.pr.read().pr().variant();
-        let reload = self.iwdg.rlr.read().rl().bits();
+        let psc = self.iwdg.pr().read().pr().variant().unwrap(); // <--- Unwrap!!
```

Comment:
Any thoughts as to what to do here? Should we provide a default or something?

Reply:
Looks like H5 guy has broke this:
https://github.com/stm32-rs/stm32-rs/commit/c812a9e90019ab314ca761cf35215a040d987e4f

There should be default 256 value

Reply:
ok

Reply:
https://github.com/stm32-rs/stm32-rs/tree/iwdg-pr

Reply:
Perfect, thanks a lot! https://github.com/stm32-rs/stm32-rs/pull/1045